browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -197,33 +197,6 @@ void BraveTabStrip::AddedToWidget() {
   }
 }
 
-SkColor BraveTabStrip::GetTabSeparatorColor() const {
-  if (ShouldShowVerticalTabs()) {
-    return SK_ColorTRANSPARENT;
-  }
-
-  Profile* profile = controller()->GetProfile();
-  if (!profile->IsRegularProfile()) {
-    if (profile->IsTor()) {
-      return SkColorSetRGB(0x5A, 0x53, 0x66);
-    }
-
-    // For incognito/guest window.
-    return SkColorSetRGB(0x3F, 0x32, 0x56);
-  }
-
-  // If custom theme is used, follow upstream separator color.
-  auto* theme_service = ThemeServiceFactory::GetForProfile(profile);
-  if (theme_service->GetThemeSupplier()) {
-    return TabStrip::GetTabSeparatorColor();
-  }
```

Comment:
I guess custom theme should be fine, as the `TabStrip::GetTabSeparatorColor()` picks blended color considering min contrast between the background.

https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/tabs/tab_strip.cc;l=2188;drc=b996220ae5827a19279d135b68d105118b466166